guides/webpack.md

```diff
@@ -0,0 +1,239 @@
+## Webpack Configuration for Torch
+
+Below is an example `webpack.config.js` (version 3.0.0) that imports Torch javascript, css,
+fonts, & images correctly.
+
+```javascript
+var path = require('path')
+var ExtractTextPlugin = require('extract-text-webpack-plugin')
+var CopyWebpackPlugin = require('copy-webpack-plugin')
+var env = process.env.MIX_ENV || 'dev'
+var isProduction = (env === 'prod')
+
+module.exports = {
+  entry: {
+    'app': ['./js/app.js', './css/app.sass']
+  },
+
+  output: {
+    path: path.resolve(__dirname, '../priv/static/'),
+    filename: 'js/[name].js'
+  },
+
+  devtool: 'source-map',
+
+  module: {
+    rules: [{
+      // SASS Loader
+      test: /\.(sass|scss)$/,
+      include: /css/,
+      use: ExtractTextPlugin.extract({
+        fallback: 'style-loader',
+        use: [
+          {
+            loader: 'css-loader',
+            options: {
+              minimize: isProduction,
+              sourceMap: true
+            }
+          },
+          {
+            loader: 'resolve-url-loader'
+          },
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceComments: !isProduction,
+              sourceMap: true
+            }
+          }
+        ]
+      })
+    }, {
+      // JS Loader
+      test: /\.js?$/,
+      include: /js/,
+      exclude: /node_modules/,
+      use: [{
+        loader: 'babel-loader',
+        query: {
+          presets: ['es2015'],
+          plugins: [],
+          cacheDirectory: true
+        }
+      }]
+    },
+    {
+      // Image Loader
+      test: /\.(png|svg|jpg|gif)$/,
+      exclude: /(fonts|node_modules)/,
+      use: [{
+        loader: 'file-loader',
+        options: {
+          name: '[name].[ext]',
+          outputPath: '../images/'
+        }
+      }]
+    },
+    {
+      // Font Loader
+      test: /\.(woff|woff2|eot|ttf|otf|svg)$/,
+      exclude: /(images|node_modules)/,
+      use: [{
+        loader: 'file-loader',
+        options: {
+          name: '[name].[ext]',
+          outputPath: 'fonts/'
+        }
+      }]
+    },
+    {
+      // Torch Image Loader
+      test: /torch.+images.+\.(png|svg|jpg|gif)$/,
+      use: [{
+        loader: 'file-loader',
+        options: {
+          name: '[name].[ext]',
+          outputPath: 'images/',
+          publicPath: '../'
```

Comment:
I might add a note saying your `name:`, `outputPath:` and `publicPath:` may need to be updated based on your final asset folder structure.